### ISSUE TITLE
Key to a Chest has empty value

### ISSUE THREAD

**Issue by AmProsius** [AmProsius/gothic-1-community-patch#180]
**Describe the bug**
The Key to a Chest has the label "Value" in the inventory, but the item has no value.

**Expected behavior**
The Key to a Chest no longer has a "Value" label.

**Additional context**
The key has a value of 0 and therefor the value isn't displayed. The German description of the key is *Truhenschlüssel*.

![ScreenShot_2021_3_5_20_40_48](https://user-images.githubusercontent.com/2817152/110170570-b3657980-7dfa-11eb-94d2-fb3a06bbaaaa.jpg)

**Comment by AmProsius** [comment]
https://github.com/AmProsius/gothic-1-community-patch/blob/5533641bc89119b14deed2552352cb783a0e5343/scriptbase/_work/Data/Scripts/Content/Items/MissionItems_3.d#L206-L210

changed to

```d
    TEXT[0]             = "Old mountain fort.";
    TEXT[1]             = "An old rusty key to a chest.";
//  TEXT[2]             = "der zu einer Truhe gehört";
    //TEXT[5]             = NAME_Value;
    //COUNT[5]            = value;
```

**Comment by szapp** [comment]
This bug will be resolved by #176.
